### ISSUE TITLE
About the code: MMoE 

### ISSUE THREAD

**Issue by HuashengPang** [tankche1/Mod-squad#7]
self.q_proj = MMoE(dim, self.att_experts, num_heads, dropout=0., concat=True)
            self.out_proj = nn.ModuleList([
                nn.Sequential(
                    nn.Linear(head_dim, dim),
                    nn.Dropout(0.)
                )
                for _ in range(num_experts)
            ])
  Where is this MMoE class defined? This class cannot be found in the project file

**Comment by tankche1** [comment]
https://github.com/UMass-Foundation-Model/Mod-Squad/issues/2
